### DISCUSSION TITLE
Uppercase text in semantic enabled repository

### DISCUSSION THREAD

**Discussion by kingcrunch** [renovatebot/renovate#20923] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

newest

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### If you're self-hosting Renovate, tell us what version of the platform you run.

newest

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

A company policy requires us to add a JIRA-id to every commit. The ticket ID is uppercase, but in semantic commit enabled repositories renovate change them to lowercase, which makes the pipeline fail (and anyway: the merge request is not mergeable) and prevents us from using automerging.

How can I add something to the commit message, which _remains_ uppercase?

### What you tried so far.

Documentation primary. `commitMessage*`-configuration options

**Comment by rarkins** [comment]
Might be blocked by #12062

Alternatively might be solved by #16839 if you can put the ticket ID in the commit scope?

**Reply by kingcrunch** [reply]
Thanks for your reply.

The commit scope is not an option currently. So, yeah, it seems it depends on #12062, which is locked "as resolved" (and open) and seems to be stale.

Is there's anything going on in this direction?

As far as I understand the only requirement set by semantic-commit messages is, that the first letter should be lowercase. I don't know, if this is even a hard requirement.

**Reply by rarkins** [reply]
It's now unlocked. We had a PR accepted for this and unfortunately it caused some side effects so we needed to roll it back. I would be looking for a community contribution if it is to move forward
